lib/StaticUniformGrid.js

```diff
@@ -0,0 +1,477 @@
+'use strict';
+var Cesium = require('cesium');
+var AxisAlignedBoundingBox = Cesium.AxisAlignedBoundingBox;
+var Cartesian3 = Cesium.Cartesian3;
+var defineProperties = Cesium.defineProperties;
+var clone = require('clone');
+var triangleAxisAlignedBoundingBoxOverlap = require('./triangleAxisAlignedBoundingBoxOverlap');
```

Comment:
When we eventually merge `ao-master` and `master` we should clean up the code to follow the Node guide, like cleaning up the imports and converting applicable callbacks and third-party functions to promises.